frontend/LoginPage.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+
+function LoginPage() {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleLogin = async () => {
+        const response = await fetch('/login', {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/json' },
+            body: JSON.stringify({ email, password })
+        });
+
+        const data = await response.json();
+        console.log(data);
+    };
+
+    return (
+        <div>
+            <input type="text" placeholder="email" onChange={(e) => setEmail(e.target.value)} />
+            <input type="password" placeholder="pass" onChange={(e) => setPassword(e.target.value)} />
+            <button onClick={handleLogin}>Login</button>
+        </div>
+    );
```

Comment:
password field should only accept integers

```suggestion
    return (
        <div>
            <input type="text" placeholder="email" onChange={(e) => setEmail(e.target.value)} />
            <input type="password" placeholder="pass" onChange={(e) => setPassword(e.target.value.replace(/\D/g, ''))} />
            <button onClick={handleLogin}>Login</button>
        </div>
    );
```